### DISCUSSION TITLE
Show coverage & best drivers

### DISCUSSION THREAD

**Discussion by ghost** [MKT-Playbook/Hub#33] [Feature requests]
It be fantastical if you could see, where the loadout misses cards for top shelf’s to know the coverage. (Coverage gaps)

also if possible, at a glance which drivers are used for the maps and you are able to sort them for items (coinbox, boomerang etc)

this could help to replace ranked drivers with better ones :-)

**Comment by TitusPosiadly** [comment]
Hey there and thanks for the ideas.
I am actually already working on exactly this feature. The coverage part is already done but I'm currently still trying to figure out a way to improve existing coverage (e.g. get better cards/higher level etc.). I will most likely get a BETA version up and running soon. Don't know if you are already in the beta but if not I would love to have you on board. Just send an email and I will send the invite once the new beta version is online.
